modules/social_features/social_group/src/Routing/RouteSubscriber.php

```diff
@@ -57,6 +58,14 @@ protected function alterRoutes(RouteCollection $collection) {
       );
     }
 
+    // Do not allow to accept invitation without "join group" permission.
```

Comment:
@rochek03 Can you explain why you added this here and not in the `social_group_invite` itself? 

If it's really needed then would be good to also add explanation here in code.